src/components/SingleReview.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getSingleReview } from "../api";
+
+function SingleReview() {
```

Comment:
there's no way for the user to navigate to a single review at the moment, can you include a link to the single reviews from the all reviews page